From 7ce11baa62224568773c22c522153acfa3e4cd7f Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 15 Mar 2016 13:59:12 +0000
Subject: [PATCH 0662/1000] MANTIS 39795 Mantis 39795: CCM Conveyor Device
 Handler Extend "ConveyorDev.idl" to support error codes INTRUSION,
 FRONT_INTRUSION, and BACK_INTRUSION

git-svn-id: svn://localhost/SelfServiceCommon/trunk@998 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index c3a507a8..840e0537 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.8";
+   const string interface_version = "1.9";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -178,7 +178,10 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       ERR_HANDLE_EXTENDED,          /**< A handle protrudes from the bag. */
       ERR_PIECE_TOO_WIDE,           /**< Piece is to wide. */
       ERR_PIECE_OVERSIZED,          /**< Piece is too big. */
-      ERR_WEIGHT_OUT_OF_RANGE       /**< Piece is too light or too heavy. */
+      ERR_WEIGHT_OUT_OF_RANGE,      /**< Piece is too light or too heavy. */
+      ERR_INTRUSION,                /**< Safety intrusions of unknown kind. */
+      ERR_FRONT_INTRUSION,          /**< Front-side safety intrusion. */
+      ERR_BACK_INTRUSION            /**< Back-side safety intrusion. */
    };
 
    /**
-- 
2.41.0.windows.1

